x-pack/plugins/security/server/lib/watch_status_and_license_to_initialize.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Observable } from 'rxjs';
+
+export function watchStatusAndLicenseToInitialize(xpackMainPlugin, downstreamPlugin, initialize) {
```

Comment:
I'll likely add some documentation to this file before we merge into `security-app-privs`, just for my own benefit. No need for you to do anything though